modules/nf-core/khmer/trimlowabund/main.nf

```diff
@@ -0,0 +1,49 @@
+process KHMER_TRIMLOWABUND {
+    tag "${meta.id}"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/khmer:3.0.0a3--py37haa7609a_2' :
+        'biocontainers/khmer:3.0.0a3--py37haa7609a_2' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("*.abundtrim"), emit: sequence
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    if (!task.memory) {
+        log.info '[KHMER_TRIMLOWABUND] Available memory not known - defaulting to 16GB. Specify process memory requirements to change this.'
+        avail_mem = 16
```

Comment:
16 is too much for a process that is set to process_low in the label

Reply:
This one is a bit odd. It is single threaded, but requires at least 16Gb in most cases.

Reply:
OK, increased it to `process_medium`